hack/install-ckcp.sh

```diff
@@ -18,14 +28,15 @@ while ! oc rsh -n ckcp deployment/ckcp ls /etc/kcp/config/admin.kubeconfig; do
   sleep 10
 done
 
-CKCP_KUBECONFIG=${CKCP_KUBECONFIG:-/tmp/ckcp-admin.kubeconfig}
-oc rsh -n ckcp deployment/ckcp sed 's/certificate-authority-data: .*/insecure-skip-tls-verify: true/' /etc/kcp/config/admin.kubeconfig > ${CKCP_KUBECONFIG}
+oc rsh -n ckcp deployment/ckcp sed 's/certificate-authority-data: .*/insecure-skip-tls-verify: true/' /etc/kcp/config/admin.kubeconfig > ${KCP_KUBECONFIG}
```

Comment:
We should not override the `KCP_KUBECONFIG` - I don't want this script to override my CPS kubeconfig that I have set for my preview mode.

Reply:
reverted back to CKCP_KUBECONFIG with instructions in command output and in readme.md